src/localization/generalStrings.json

```
@@ -1,5 +1,8 @@
 {
   "gb": {
+    "enter_patient_details": "Enter the patients details to start searching for them!",
```

:nit 
```suggestion
    "enter_patient_details": "Enter the patient's details to start searching for them!",
```
The details belong to the patient, so there should be an apostrophe, I think.